gallery/src/components/NxDropdownIconOnly/NxDropdownIconOnlyPage.tsx

```diff
@@ -0,0 +1,169 @@
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-dropdown-button</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                <NxCode>&lt;a&gt;</NxCode> and <NxCode>&lt;button&gt;</NxCode> elements
+                within the dropdown menu
+              </NxTable.Cell>
+              <NxTable.Cell>
+                Styles the buttons and links as clickable rows within the menu
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-dropdown-link</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                <NxCode>&lt;a&gt;</NxCode> elements within the dropdown menu
+              </NxTable.Cell>
+              <NxTable.Cell>
+                Applies typical blue styling to links within the menu
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-dropdown-right-button</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                An additional icon-only button which can appear on the right side of a menu row
+              </NxTable.Cell>
+              <NxTable.Cell>
+                Sometimes it is desirable to add a icon-only button to a menu row which performs an action distinct
+                from clicking on the row itself - for instance, a trashcan button which deletes the row. For this
+                scenario, the icon button, a preceding sibling of the main row button, must include this class and{' '}
+                <NxCode>nx-dropdown-button-content</NxCode> must be used in conjunction.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-dropdown-button-content</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Wrapper around text content of menu rows
+                when <NxCode>nx-dropdown-right-button</NxCode> is in use.
+              </NxTable.Cell>
+              <NxTable.Cell>
+                This wrapper is necessary around the text content of
+                any <NxCode>nx-dropdown-button</NxCode> which is in a menu containing
+                any <NxCode>.nx-dropdown-right-button</NxCode>s.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Navigation Example"
+                        liveExample={NxDropdownIconOnlyNavigationExample}
+                        codeExamples={nxDropdownIconOnlyNavigationExampleCode}>
+      An example of an <NxCode>NxDropdown</NxCode> as it might be used to implement a navigation list.
+      Note that the menu can contain either <NxCode>&lt;a&gt;</NxCode> or
+      {' '}<NxCode>&lt;button&gt;</NxCode> elements; this example contains both.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Links Example"
```

Comment:
Is it valuable to have both of these examples on this page?

Reply:
> Is it valuable to have both of these examples on this page?

I debated this. I think it's simpler with one, consolidated here: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/7b3af6212d9d860691d028159220461d0d37da6e